README.md

```diff
@@ -10,37 +10,6 @@ The common scenario is:
 1. Launch Firefox and open this tool twice in two tabs.
 2. Select the second tab and connect (to the first one).
 
-
-Source Directory Structure
---------------------------
-Since this app loads source modules directly from Firefox source directory,
-paths must be properly set.
-
-Here is the default supported directory structure.
-(if you use the same directory alignment you don't need to setup anything)
-
-
-```
-<root>
-  - mozilla.org
-    - fx-team
-      + devtools
-  - github.com
-    - janodvarko
-      + rep-tester
-```
-
-* `devtools` Root directory for Firefox built-in devtools
-* `rep-tester` Root directory for this tool
-
-Webpack config is using the following alias (see `webpack.config` file):
-
-`devtools" => config.firefoxSrc + "/devtools"`
-
-This allows requiring devtools modules (directly from Firefox source dir) as follows:
-
-`require("devtools/client/shared/components/reps/grip");`
-
```

Comment:
Also the info saying that webpack is actually using the `firefoSrc` field + an example of require call could be also moved into the Setup section. `require("devtools/client/shared/components/reps/grip");`